releases/release-1.27/README.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Kubernetes 1.27 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.27
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.27/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.27/release-team.md)
+* [Meeting Minutes](http://bit.ly/k8s127-releasemtg) (members of [release-team@] receive meeting invites)
+* [v1.27 Release Calendar][k8s127-calendar]
+* Contact: [#sig-release] on slack, [release-team](mailto://release-team@kubernetes.io) on e-mail
+* [Internal Contact Info] (accessible only to members of [release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](https://bit.ly/k8s127-enhancements)
+* [Feature blog Tracking Board](#)
+* [Bug Triage Tracking Sheet](https://bit.ly/k8s127-bug-triage-tracking)
+* [CI Signal Project Board & Issue Tracking](#)
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.27 milestone](https://github.com/kubernetes/kubernetes/milestone/60)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## Summary
+
+The 1.27 release cycle is as follows:
+
+- **Monday 9th January 2023**: Week 1 — Release cycle begins
+- **Thursday 2nd February 2023**: Week 4 — [Production Readiness Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
+- **[01:00 UTC Friday 10th February 2023 / 18:00 PDT Thursday 9th February 2023](https://everytimezone.com/s/cc15bd3a)**: Week 5 — [Enhancements Freeze](../release_phases.md#enhancements-freeze)
+- **[01:00 UTC Wednesday 15th March 2023 / 17:00 PDT Tuesday 14th March 2023](https://everytimezone.com/s/c4971746)**: Week 10 — [Code Freeze](../release_phases.md#code-freeze)
+- **[01:00 UTC Wednesday 22nd March 2023 / 17:00 PDT Tuesday 21st March 2023](https://everytimezone.com/s/5d64ada0)**: Week 11 — [Test Freeze](../release_phases.md#test-freeze)
+- **Tuesday 4th April 2023**: Week 13 — Docs must be completed and reviewed
+- **Thursday 13th April 2023**: Week 14 — Kubernetes v1.27.0 released
```

Comment:
Maybe stick KubeCon on the end, so people can be clear and understand that KubeCon EU is the week after.